### DISCUSSION TITLE
Ionic iOS Native App has no referrer

### DISCUSSION THREAD

**Discussion by dziedziela** [Web3Auth/Web3Auth#394] [Help]
When trying to authenticate using Web3Auth from a native iOS app built in Ionic, it shows a blank screen and the error said "redirect url is not on origin". What Whitelisted URLs shall we add to overcome this issue?

**Comment by dhruvTor** [comment]
Hey @dziedziela your whitelist url will be your bundle Identifier for your iOS app followed by ://auth so something like this (xyz://auth) where xyz is your bundle Identifier which can be found in Targets -> General

**Reply by dziedziela** [reply]
Thank you @dhruvTor we tried and didn't seem to work.

**Reply by dhruvTor** [reply]
Hey @dziedziela is the auth page opening inside the app or taking to you to an external browser ?

**Reply by dziedziela** [reply]
yes, it does open an external browser @dhruvTor

**Reply by dhruvTor** [reply]
Please try adding your callback scheme in your url schemes in your ios app
